packages/components/organisms/f-checkout/src/services/addressGeocodingApi.js

```diff
@@ -0,0 +1,17 @@
+import axios from 'axios';
```

Comment:
Just to confirm, is the name of the API `addressGeocoding`? (I don't know, that's why I'm asking, hehe).

Reply:
yep it is indeed :)